src/plugins/es_ui_shared/static/forms/hook_form_lib/hooks/use_form.ts

```diff
@@ -25,6 +25,7 @@ import { createArrayItem, getInternalArrayFieldPath } from '../components/use_ar
 const DEFAULT_OPTIONS = {
   valueChangeDebounceTime: 500,
   stripEmptyFields: true,
+  stripUnsetFields: false,
```

Comment:
Cna we document this and what it does in https://github.com/elastic/kibana/tree/main/src/plugins/es_ui_shared/static/forms/docs as well?

Reply:
Good call! I added some documentation with [99b7023](https://github.com/elastic/kibana/pull/194148/commits/99b702380ea2638475164ce9dec43a1e81f3f9ef).